ocs/ocp.py

```diff
@@ -154,3 +154,20 @@ def wait_for_resource_status(
             if sample.status.phase == condition:
                 return True
         return False
+
+    # this method location is temporary, will be changed to ocs class
+    def exec_ceph_cmd(self, ceph_cmd):
+        """
+        Execute a Ceph command on the Ceph tools pod
+
+        Args:
+            ceph_cmd (str): The Ceph command to execute on the Ceph tools pod
+
+        Returns:
+            str: Ceph command output in a Json format
+        """
+        pods_list = shlex.split(self.get(out_yaml_format=False))
```

Comment:
What will happen if I will call this func with OCP object that is initialized for a resource that is not a "pod"?
Also, I think this function shouldn't be under OCP object but this can stay for now until we'll have better design and hierarchy

Reply:
@RazTamir you're right. My initial intention was to puth this function in a location that will instantiate an ocp object of kind pod.
For now, I'll do it in this same file and I'll move it to a more appropriate location later.